### DISCUSSION TITLE
Questions and Suggestions

### DISCUSSION THREAD

**Discussion by elielbony** [prasathmani/tinyfilemanager#603] [Q&A]
great manager! Congratulations!

Have a question why the latest version doesn't have the option to change **permission??????**

A surge of better ones would be wonderful to see a folder tree on the left side.

**Comment by prasathmani** [comment]
The change Permissions option is available, check in demo.
Folder tree on the left side - in earlier version it was there, due to performance impact it is removed.

**Reply by elielbony** [reply]
Hi, Sorry to bother you again, regarding your quick response...
I checked all the previous versions and none of them exist the "**tree**" menu all are default...
Would it be possible to make a version available for download with the left tree menu?

Another thing is that none of the versions works the display of "**Hide Perms/Owner columns**" in the settings page "**Settings**" we can change it to "on/off" and it changes in the source code too, but it doesn't display the column at all.

How much does your services cost to repair it, and let it download in new version?
I believe there are more users with the same problem.

Thank you, Best regards...

**Reply by prasathmani** [reply]
Permissions column only available in Linux based system, in windows machine it will not support.
attached older version which support tree-view.
[tinyfilemanager-0.9.php.zip](https://github.com/prasathmani/tinyfilemanager/files/6868135/tinyfilemanager-0.9.php.zip)

**Comment by modcolo** [comment]
please add pagination

![screencapture](https://user-images.githubusercontent.com/76621038/137303829-830f719e-7cdd-4f0a-aa64-153250fe7d12.png)
